### DISCUSSION TITLE
Meeting Not Found from imported source

### DISCUSSION THREAD

**Discussion by jimmedlen** [code4recovery/12-step-meeting-list#1387] [General]
Meetings List shows error message "Meeting Not Found" for meetings from external source.

Area22indiana.org and aaDistrict4243.com overlap. I added the source from aadistrict4143 to area22indiana.org.
Duplicate meeting entries were saved to 'Drafts' so only the external source meetings are being displayed.
When searching for the meetings in a location from the external source the meetings are listed but there is an error message
"Meeting Not Found"

**Comment by SIA-WEBSUP** [comment]
can you give an example of a meeting in this category?

can you provide the URL to the json feed for the 'external' source ( so we can be sure the meetings are there and well formed )?
can you provide the URL for to your json feed?

**Comment by andrewp123-bit** [comment]
@SIA-WEBSUP  I'm getting something similar but it's not for meetings in the same area. 

For example, this meeting shows as published and active [1], but when I click the live link, I get the "Meeting Not Found" error [2]. 

[1]
![b121a514a51d936df455d64691e51c50](https://github.com/user-attachments/assets/f76f61e3-7959-448f-95dd-a1530acb6a52)
[2]
![ea22193775f8dbf6a6137e43ff51616e](https://github.com/user-attachments/assets/30b525da-c602-4891-b78f-b9528406a023)

The only thing I've noticed is this is the case for **all** meetings that have times (i.e the only meetings displaying right now are those set to "appointment". I'm fairly confident it's not something with my broader wordpress settings, like the time format, as those are defined. 

Would appreciate any help! I think this is the last thing before I'm fully up and running, which is remarkable. 

Thank you all.

**Reply by joshreisner** [reply]
i'm pretty sure this is a timezone issue. on your meetings > settings page, do have a timezone set? if not that's ok, but it means you need to set a timezone on every individual meeting location. 

<img width="524" alt="Screenshot 2025-05-09 at 6 59 09 PM" src="https://github.com/user-attachments/assets/8b7683d2-ae5f-4520-97ef-315da3de3b36" />

<img width="1137" alt="Screenshot 2025-05-09 at 6 59 53 PM" src="https://github.com/user-attachments/assets/06f6278d-a1d8-4152-90c3-1f82edcabde3" />

**Comment by andrewp123-bit** [comment]
Dear @joshreisner, I don't have a timezone set on the main settings page (As we have meetings all across the world), but I do have one set on the international meeting. It was blank before, so I can go ahead and change those, but the issue persists: 
![d6ddcf6556c93d1555690db7d7d9f652](https://github.com/user-attachments/assets/9433bda1-0338-4e0a-b0d8-5b59a59228de)

Same errror: "Meeting not found"

**Reply by andrewp123-bit** [reply]
I'm going through and adding the timezone for each listing, I'm thinking I need to have it listed for **every** meeting so they all flow together. 

Interestingly the ones that are listed don't have a timezone (but they also are all listed as appointment!)

Will report back!
